server/services/store/sqlstore/migrations/migrations_files/000011_match_collation.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Nothing to be done here
+-- This page is intentionally left blank
```

Comment:
You added an empty query in the `.up.sql` file because if not, it fails, I guess it should be the same here.

Reply:
Ah, great catch! Fixed now.